### PR TITLE
[#52529] Nextcloud integration admin guide not helpful in chosing a stable Nextcloud version

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/work_packages/52529

### CODE REVIEW

**Review by MayaBerd — APPROVED** [review]
@wielinde thanks for the quick update and differentiating between latest and latest stable versions! 
I've tweaked the wording slightly and pushed the changes directly.

**Comment by MayaBerd** [comment]
@as-op after this is PR is through, could you please check if any links may be broken? One of the existing headings will be changed.

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 1]
```diff
@@ -27,15 +27,21 @@ OpenProject offers close integration with Nextcloud to allow users to:
...
+- Nextcloud version in the latest `stable` version. 
+  - If you run Nextcloud in the community edition be careful to not 
+    blindly follow the update hints in the administration area of a Nextcloud instance, as they nudge you to use the
+    `latest` version, which might not be the latest `stable` version. We recommend to check the [tags of Nextcloud's official
+    Docker image](https://hub.docker.com/_/nextcloud). There you can see which version is currently the stable one. 
+  - If you installed Nextcloud via Docker then we advice you to pin it to a `stable` tag.
 
-We recommend using the latest versions of both Nextcloud and OpenProject to be able to use the latest features. 
+
+We recommend using the latest stable versions of both Nextcloud and OpenProject to be able to use the latest features.
```
Comment (wielinde) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: @MayaBerd I removed that line on purpose as that is a duplication of the text above the bullet list.

**Comment by as-op** [comment]
@MayaBerd There was only one link in the TOC above, I adjusted it. 
No links from the website or other docs.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 1 above)
Comment (wielinde) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Will you remove it again?

Comment (MayaBerd) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: @wielinde yes, just did

**Review by machisuji — APPROVED** [review]
👍

**Pull request merged**